doc/quickstart/otb_quickstart.rst

```diff
@@ -13,42 +14,43 @@
 
 ORFEO Toolbox library (OTB) is a high performance library for image processing targeted on remote sensing.
 
-This Quick Start describes how to:
+This Quickstart describes how to:
 
-  * Get metadata informations from an image
+  * Get metadata information from an image
   * Perform mathematical operations between image bands
   * Open raster images with the application monteverdi, perform segmentation (mean-shift clustering) and visualize the result
   * Perform supervised classification based on Support Vector Machine algorithm
 
-The OTB applications provide lot's of interesting tools which facilitate the manipulation of images. All these tools are available through:
+The OTB applications provide lots of interesting tools which facilitate the manipulation of images. All these tools are available through:
 
-  * CLI (command line interface) : all applications can be called from a terminal starting with `otbcli_` plus the application name
-  * GUI (a standalone graphical user interface in Qt) : the applications can be called from a terminal with the alias `otbgui_` plus the application name. An other option is to use a small launcher (available in the menu -> Geospatial -> Spatial Tools -> OTB Launcher)
-  * QGIS plugin : available through the processing framework
+  * CLI (command line interface). All applications can be called from a terminal starting with `otbcli_` plus the application name.
+  * GUI (a standalone graphical user interface in Qt). The applications can be called from a terminal with the alias `otbgui_` plus the application name. Another option is to use a small launcher (available in the :menuselection:`menu --> Geospatial --> Spatial Tools --> OTB Launcher`).
+  * QGIS plugin - available through the processing framework.
 
 Sample data used in this quickstart can be found in :
   * /home/user/data/north_carolina/rast_geotiffs
 
 .. contents:: Contents
+   :local:
 
-Display metadata informations in an image 
-================================================================================
+Display metadata information in an image 
+========================================
 
-You can get all the metadata informations contained in an image with the command : `otbcli_ReadImageInfo`
+You can get all the metadata information contained in an image with the command : `otbcli_ReadImageInfo`
 The unique parameter is the Input image file name, for example::
 
   otbcli_ReadImageInfo -in ortho_2001_t792_1m.tif
```

Comment:
I got a 'file does not exist' message on this so couldn't proceed.

Reply:
I can see that there is a file "lsat7_2002_10.tif" use in the quickstart for the BandMath application.

Can you try to replace 'ortho_2001_t792_1m.tif' by 'lsat7_2002_10.tif' ?

Hope it helps